### DISCUSSION TITLE
What is a group of flamingos called?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#563] [Galaxy Trivia] [answered]
🦩🦩🦩

**Comment by peterbaccaro** [comment]
Flamingos have several collective nouns, the main ones being **a flamboyance**, **a stand**, and **a colony**. Other collective nouns for Flamingos include a **regiment**, **a flurry**, and **a skein**. 

The collective noun **flamboyance** means to show off, a fitting term for large groups of bright pink flamingos.

**Comment by thuongtruong1009** [comment]
A group of flamingos is called a "flamboyance".

**Comment by SebastienXia** [comment]
flamboyance

**Comment by SumitKumarKar01** [comment, marked as answer]
flamboyance

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/SumitKar01?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉
